src/api/model/my-account-personal-data.ts

```diff
@@ -0,0 +1,4 @@
+export interface MyAccountPersonalData {
```

Comment:
Please squash commits into one if github does not provide it automatically when merge